packages/desktop-client/src/components/reports/ReportTopbar.tsx

```diff
@@ -12,16 +14,19 @@ import {
   SvgListBullet,
   SvgQueue,
   SvgTag,
+  SvgCamera,
 } from '../../icons/v1';
 import { SvgChartArea } from '../../icons/v1/ChartArea';
 import { theme } from '../../style';
+import { SnapshotButton } from '../common/SnapshotButton';
 import { SpaceBetween } from '../common/SpaceBetween';
 import { View } from '../common/View';
 import { FilterButton } from '../filters/FiltersMenu';
 
 import { GraphButton } from './GraphButton';
 import { SaveReport } from './SaveReport';
 import { setSessionReport } from './setSessionReport';
+import { getToday } from './util';
```

Comment:
monthUtils would be better for this, you can grab it through loot core.

```suggestion
import * as monthUtils from 'loot-core/shared/months';
```